Logger/LogFactory.cs

```diff
@@ -2,10 +2,24 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? FilePath;
```

Comment:
Fields are typically prefixed with an underscore. Though in general we favor using fields rather than fields when possible.
```suggestion
        private string? FilePath { get; }
```